coap/README.md

```diff
@@ -0,0 +1,83 @@
+# libcoap: A C implementation of the Constrained Application Protocol (RFC 7252)
```

Comment:
This could be soft link to `README.md` file from libcoap. Please check `expat` component for this.